assets/app/scripts/directives/events.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .directive('events', function($routeParams, $filter, DataService, ProjectsService, Logger) {
+    return {
+      restrict: 'E',
+      scope: {
+        resourceKind: "@",
+        resourceName: "@",
+        parentScope: "="
```

Comment:
> was also using the build.metadata.annotations for the Build events, but I can pass that separately

You were using it to get the build pod name, right? Just pass the name of the pod.

```
<events kind="Pod" name="ruby-sample-build-17-build"></events>
```